MeiliSearchBox.vue

```diff
@@ -72,9 +75,14 @@ export default {
     },
 
     update(options) {
+      this.updateTheme(options)
       this.$el.innerHTML =
         '<input id="meilisearch-search-input" class="search-query">'
       this.initialize(options)
+    },
+
+    updateTheme(options) {
+      this.theme = options.enableDarkMode === true ? 'dark' : options.enableDarkMode === false ? 'light' : 'auto'
```

Comment:
If the value is set to `auto`  it is considered `true` and thus it will always showcase the dark mode.

I changed your ternary nesting with if/else for readability
```suggestion
    updateTheme(options) {
      if (options.enableDarkMode) { // will enter here even if it is auto ?
         this.theme = 'dark'
      }
      else if (!options.enableDarkMode) {
         this.theme = 'light'
      }
   }
```